tests/standalone_tests.sh

```diff
@@ -34,8 +35,8 @@ else
 fi
 parametrizations_arr=($parametrizations)
 
-# tests to skip - space separated
-blocklist='tests/profiler/test_profiler.py::test_pytorch_profiler_nested_emit_nvtx'
+# TODO: tests to skip - space separated
+blocklist='tests/profiler/test_profiler.py::test_pytorch_profiler_nested_emit_nvtx tests/utilities/test_warnings.py'
```

Comment:
Why are we ignoring this?

Reply:
Trying to find why the stand alone does not find tests...

Reply:
Oh yeah, just realized we call it manually below